integration/vm_factory/vm_cache_test.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+source "${cidir}/../../metrics/lib/common.bash"
+
+kata_runtime=$(command -v kata-runtime)
+
+# Environment variables
+KATA_CACHE_INIT="${kata_runtime} factory init"
+KATA_CACHE_STATUS="${kata_runtime} factory status"
+KATA_CACHE_DESTROY="${kata_runtime} factory destroy"
+KATA_CACHE_SOCK="/var/run/kata-containers/cache.sock"
+
+IMAGE="${IMAGE:-busybox}"
+TEST_PLAYLOAD="2"
+CONTAINER_NAME=$(random_name)
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+
+setup() {
+        extract_kata_env
+}
+
+teardown(){
```

Comment:
`disable_vm_cache_config()` is called by the end of `test_vm_cache_factory_init_destroy()`. Shouldn't make sense to call it here as well? In case something goes wrong between the time you enabled it and ran the test.